angular-konami.js

```diff
@@ -60,4 +60,4 @@ angular.module('konami', [])
         scope.$on('$destroy', stopListening);
       }
     };
-  }]);
\ No newline at end of file
+  }]);
```

Comment:
I was about to explain why we should use newlines at the end of files when I understood you were not removing it, but placing. Thanks ;)